sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/CombineFns.java

```diff
@@ -588,6 +597,13 @@ public CoCombineResult extractOutput(Object[] accumulator, Context c) {
       }
       return new ComposedAccumulatorCoder(coders);
     }
+
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      for (CombineFnWithContext combineFn : combineFnWithContexts) {
+        builder.include(combineFn);
```

Comment:
Good catch. This ends up being tricky; the current behavior is to throw if a duplicate display item is registered (in most scenarios this would be a mistake).

I've updated the logic for CombineFns to conditionally add a `$n` to the namespace if more than one combineFn have the same type.